include/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/Kepler/BrouwerLyddaneMean.hpp

```diff
@@ -0,0 +1,99 @@
+/// Apache License 2.0
```

Comment:
I might be wrong. In my mind, Classical Orbital Elements are Kepler Orbital Elements 🤔

Reply:
> We can still inherit from COE. 

Yeah I think we should still do this. It makes sense to me to add a new folder yup: Kepler: unperturbed two-body dynamics and Brouwer is averaged method taking into account perturbations of central body gravity. I guess we have J2, J3 in Kepler so that's not really true indeed...

Reply:
Yeah... I've made a separate folder for BrouwerLyddaneMean for now. I think we can refactor into an OrbitalElement folder later if we decide to add more elements.